src/Numerics.Tests/DistributionTests/Continuous/BetaScaledTests.cs

```diff
@@ -85,6 +85,7 @@ public void BetaScaledCreateFailsWithBadParameters()
         [Test]
         public void ValidateToString()
         {
+            System.Threading.Thread.CurrentThread.CurrentCulture = System.Globalization.CultureInfo.InvariantCulture;
             var n = new BetaScaled(1d, 2d, 0.0, 1.0);
             Assert.AreEqual("BetaScaled(α = 1, β = 2, μ = 0, σ = 1)", n.ToString());
```

Comment:
Just curious, are there really cultures where floating point numbers with 0 decimal part and a single digit only are rendered differently than the expected string?

Reply:
I have fixed the tests failing on my machine but looking at it, that does seem weird. Let me try to run the test again.

Reply:
Looks like I "fixed" one un-needed test or I had an unrelated issue with this test and I applied a wrong fix. Anyways I'll undo this one! Thanks for the catch!